docs/templates/modules/export.md

```diff
@@ -0,0 +1,46 @@
+![Status: Not started](https://img.shields.io/badge/status-not%20started-red) &nbsp;<sup>→</sup>&nbsp;
+[![Go to issue](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/1)](https://github.com/microsoft/cloud-hubs/issues/1)
+
+# export.bicep
+
+This module deploys a **Cost Management export** instance for any scope.
+
+Note this export can be used for any purpose and does not have to be exclusively for FinOps hubs, but hubs do require some specific settings. See [Required settings for FinOps hubs](#required-settings-for-finops-hubs) for details.
+
+On this page:
+
+- [Parameters](#parameters)
+- [Resources](#resources)
+- [Outputs](#outputs)
+
+---
+
+## Parameters
+
+- **exportName** (string) – Name of the export instance.
+- **amortize** (boolean) – Optional. Indicates whether the export should amortize reservation purchases. Default: `false`.
```

Comment:
The warning about amortized implies this would not be optional or false for RI. Please clarify.

Reply:
My plan was to use this module as a general-purpose way to automate export creation. It's not exclusively for hubs. `hub.bicep` forces amortization to be enabled for hubs.